### ISSUE TITLE
Define a technical solution for supporting sub-national level datasets

### ISSUE THREAD

**Issue by pzwsk** [GFDRR/open-risk-data-dashboard#445]

**Comment by robertorudari** [comment]
HI Pierre. I agree with this first approach that seems also impacting the overall structure of the tool very little. However, it is still unclear to me how to account this in the scoring. Will the subnational dataset appear in the stack bar at the same level as a national dataset? That would be an easy way to do it, but I think It would introduce a strong bias when one looks to the explore country page. My proposition would be to have a counter for the sub-national datasets and when a sub-national location is under analysis (I'm exploring Paris - Ile de France) I have a a view of the full evaluation of the dataset in the same way we have for the countries.

**Comment by stufraser1** [comment]
I agree with the approach. However, note that we are looking to soon update TH using the WB official naming and boundaries (modified version of GAUL 2015) available at:
https://datacatalog.worldbank.org/dataset/world-subnational-boundaries
(requires WB login to view)
